datafusion/core/tests/sqllogictests/test_files/array.slt

```diff
@@ -864,17 +864,18 @@ select make_array(['a','b'], null);
 
 ## array_append (aliases: `list_append`, `array_push_back`, `list_push_back`)
 
+# TODO: array_append with NULLS
 # array_append scalar function #1
-query ?
```

Comment:
Is this a regression in functionality (aka something that used to work no longer does)? Is it hard to fix prior to merging this PR?